### DISCUSSION TITLE
能不能出个使用视频啊？

### DISCUSSION THREAD

**Discussion by elarbor** [JackySoft/marsview#49] [General]
比如以搭建OA系统为例，出一套完整的搭建过程视频

**Comment by JackySoft** [comment]
http://docs.marsview.cc/home.html

可以先看一下这个文档，里面有项目介绍视频、页面介绍视频、综合案例视频。  也可以直接在群里问我，我24h解答。

**Reply by elarbor** [reply]
好的
